spec/controllers/api/testing/teams_controller_spec.rb

```diff
@@ -86,7 +86,7 @@
           .and(change(Parent, :count).by(-4))
           .and(change(Patient, :count).by(-3))
           .and(change(PatientSession, :count).by(-3))
-          .and(change(VaccinationRecord, :count).by(-11))
+          .and(change(VaccinationRecord, :count).by(-9))
```

Comment:
Do we know why we're no longer deleting these 2 records? I'm guessing their batch no longer belongs to a team?